test/integration/mig_stateful/controls/mig_stateful.rb

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 Google LLC
```

Comment:
Are you aware of the latest golang based [testing framework](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/infra/blueprint-test) for TF modules?  Example usage [here](https://github.com/terraform-google-modules/terraform-google-sql-db/tree/master/test/integration).

We're trying to discourage the use of kitchen and want people to write tests based on this new framework especially for new tests. Is it possible for you to switch this over to that?

Reply:
Thanks for the pointers. I'll update it accordingly